script/windows-installer/inno-setup-git-lfs-installer.iss

```diff
@@ -15,7 +17,7 @@
 AppId={{286391DE-F778-44EA-9375-1B21AAA04FF0}
 AppName={#MyAppName}
 AppVersion={#MyAppVersion}
-;AppVerName={#MyAppName} {#MyAppVersion}
+AppCopyright=GitHub, Inc. and Git LFS contributors
```

Comment:
I think the fact that it's in our license file means it's a-ok. I consider it the source of truth over this `iss` file. Can you verify that it compiles, and that the file properties show it so we can approve and merge this?